raft/rafttest: TestPause failed

raft/rafttest.TestPause [failed](https://tanzanite.cluster.engflow.com/invocations/default/d8f57495-6363-4ad7-a328-2a423f7cf47d?testReportRun=25&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvcmFmdC9yYWZ0dGVzdDpyYWZ0dGVzdF90ZXN0) on release-24.3 @ [4cbedefd790c75cb0f21f77ed8d917c8528a7d15](https://github.com/cockroachdb/cockroach/commits/4cbedefd790c75cb0f21f77ed8d917c8528a7d15):


```
raft2024/10/19 04:19:49 INFO: 3 received MsgVoteResp from 2 at term 2
raft2024/10/19 04:19:49 INFO: 3 has received 3 MsgVoteResp votes and 2 vote rejections
raft2024/10/19 04:19:49 INFO: 3 became leader at term 2
raft2024/10/19 04:19:49 INFO: 3 [logterm: 2, index: 6, vote: 3] ignored MsgVote from 4 [logterm: 1, index: 5] at term 2: supporting fortified leader 3 at epoch 1
raft2024/10/19 04:19:49 INFO: 2 [term: 2] received a MsgVote message with higher term from 4 [term: 3]
raft2024/10/19 04:19:49 INFO: 2 became follower at term 3
raft2024/10/19 04:19:49 INFO: 2 [logterm: 1, index: 5, vote: 0] cast MsgVote for 4 [logterm: 1, index: 5] at term 3
raft2024/10/19 04:19:49 INFO: 1 [term: 2] received a MsgVote message with higher term from 4 [term: 3]
raft2024/10/19 04:19:49 INFO: 1 became follower at term 3
raft2024/10/19 04:19:49 INFO: 1 [logterm: 1, index: 5, vote: 0] cast MsgVote for 4 [logterm: 1, index: 5] at term 3
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 received MsgVoteResp from 5 at term 3
raft2024/10/19 04:19:49 INFO: 4 has received 2 MsgVoteResp votes and 0 vote rejections
raft2024/10/19 04:19:49 INFO: 4 received MsgVoteResp from 1 at term 3
raft2024/10/19 04:19:49 INFO: 4 has received 3 MsgVoteResp votes and 0 vote rejections
raft2024/10/19 04:19:49 INFO: 4 became leader at term 3
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 3 [term: 2] received a MsgFortifyLeader message with higher term from 4 [term: 3]
raft2024/10/19 04:19:49 INFO: 3 became follower at term 3
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 2 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: found conflict at index 6 [existing term: 2, conflicting term: 3]
raft2024/10/19 04:19:49 INFO: replace the unstable entries from index 6
raft2024/10/19 04:19:49 INFO: 5 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 4 [term: 3] ignored a MsgFortifyLeader message with lower term from 3 [term: 2]
raft2024/10/19 04:19:49 INFO: 1 [term: 3] ignored a MsgApp message with lower term from 3 [term: 2]
    node_test.go:126: commits failed to converge!
I241019 04:19:54.853300 347 (gostd) node.go:162  [-] 13  raft.1: stop
I241019 04:19:54.853411 349 (gostd) node.go:162  [-] 14  raft.2: stop
I241019 04:19:54.853481 351 (gostd) node.go:162  [-] 15  raft.3: stop
I241019 04:19:54.853522 353 (gostd) node.go:162  [-] 16  raft.4: stop
I241019 04:19:54.853565 355 (gostd) node.go:162  [-] 17  raft.5: stop
--- FAIL: TestPause (5.38s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=25</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #132205 raft/rafttest: TestPause failed [C-bug C-test-failure O-robot P-1 T-kv branch-master]
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPause.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-43390

--- a/pkg/raft/rafttest/node.go
+++ b/pkg/raft/rafttest/node.go
@@ -49,7 +49,7 @@ func startNode(id raftpb.PeerID, peers []raft.Peer, iface iface) *node {
 	st := raft.NewMemoryStorage()
 	c := &raft.Config{
 		ID:                        id,
-		ElectionTick:              10,
+		ElectionTick:              50,
 		HeartbeatTick:             1,
 		Storage:                   st,
 		MaxSizePerMsg:             1024 * 1024,

